entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+python3 manage.py collectstatic --no-input
+python3 manage.py makemigrations
+python3 manage.py migrate
+gunicorn config.wsgi -b 0.0.0.0:8000 --workers 3 --timeout 600
```

Comment:
Try the following thing and then specify your command in `docker-compose`. In this way you don't need two entry points for development and production.
```suggestion
exec "$@"
```